client/src/entry/analysis/menu.js

```diff
@@ -193,25 +193,23 @@ export function fetchMenu(options = {}) {
             tooltip: _l("Account and saved data"),
```

Comment:
I think every item in the list is inheriting the tooltip and it is not very helpful:

![image](https://user-images.githubusercontent.com/46503462/235085734-3f8e7db6-f92b-4813-a0a3-7de50443d92f.png)

Would be nice to fix it too as part of this great enhancement :star_struck:

Reply:
This seems to be covered by #15995

Reply:
Indeed! I missed that one, thanks!